cifar/README.md

```diff
@@ -0,0 +1,31 @@
+# CIFAR and ResNets
+
+* This example shows how to run ResNets on CIFAR10 dataset, in accordance with the original [paper](https://arxiv.org/abs/1512.03385).
+* Also illustrates how to use `mlx-data` to download and load the dataset.
+
+
+## Pre-requisites
+* Install the dependencies:
+
+```
+pip install -r requirements.txt
+```
+
+## Running the example
+Run the example with:
+
+```
+python main.py
+```
+
+By default the example runs on the GPU. To run on the CPU, use: 
+
+```
+python main.py --cpu_only
+```
+
+For all available options, run:
+
+```
+python main.py --help
+```
```

Comment:
Could you add a section on results. And a little table with some train / valid values over the course of training + the final test set accuracy. Also an approximate images / sec would be helpful so people know it's running at an expected speed.

Reply:
I have a base M1 MacBook, so I cannot run full training (usually ~330 epochs with SGD). But sure, I'll add some sample logs and final accuracy value for the amount of epochs I do run.